ffmpeg/api_test.go

```diff
@@ -1244,6 +1244,54 @@ func TestTranscoder_AudioOnly(t *testing.T) {
 	audioOnlySegment(t, Software)
 }
 
+func detectionFreq(t *testing.T, accel Acceleration) {
```

Comment:
Wouldn't this test always fail CI right now since there is no model available? I guess we can make sure the test can be run locally and just comment it out.

Reply:
Good idea - note to future-self, move this and the keyframe test in a separate file with `//+build ignored`